prometheus_client/bridge/newrelic.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import newrelic.agent
+
+from .. import core
+
+
+@newrelic.agent.data_source_factory(name='Prometheus')
+class PrometheusDataSource(object):
+    def __init__(self, settings, environ):
+        self.registry = settings.get('registry', core.REGISTRY)
```

Comment:
I'm not sure it's possible to reasonably use this setting, I suspect you should drop and leave other registries to the push method.